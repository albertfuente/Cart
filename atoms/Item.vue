<template>
  <div class="item-atom">
    {{ label }}
  </div>
</template>

<script>


export default {
  name: "Item",

  props: {
    label: {
      type: String,
      default: "label",
    },
  },
};
</script>
<style scoped>
</style>