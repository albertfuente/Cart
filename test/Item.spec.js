import { shallowMount } from '@vue/test-utils'
import Item from './../atoms/Item.vue'

describe('Item.vue/Item', () => {
  it('mounts the component and creates a snapshot inside the __snapshots__ folder', () => {
    const wrapper = shallowMount(Item)
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('is rendering main div with correct class', () => {
    const wrapper = shallowMount(Item)
    expect(wrapper.classes()).toContain('item-atom')
  })

})