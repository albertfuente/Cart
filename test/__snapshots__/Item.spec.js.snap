// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item.vue/Item mounts the component and creates a snapshot inside the __snapshots__ folder 1`] = `
"<div class=\\"item-atom\\">
  label
</div>"
`;
